### PR TITLE
Integrate bootloader with OTAS program code

### PR DESCRIPTION
**Pull request by AbbyWolf-ADI**
Update OTAS makefiles to automatically compile and flash the bootloader application

### CODE REVIEW

**Review by kevin-gillespie — CHANGES_REQUESTED** [review]
I think we can add something like this to the client project.mk to build the .bin file without the bootloader section.

```bash
# Create binary output file without the bootloader
%.bin.noboot: %.elf
	@if [ 'x${VERBOSE}' = x ];                                                   \
	 then                                                                        \
	     echo "Creating ${@}";                                                   \
	     echo "Excluding bootloader section";                                    \
	 else                                                                        \
	     echo ${OBJCOPY} -O binary --remove-section .bootloader $(call fixpath,${<}) $(call fixpath,${@});    \
	 fi
	@$(OBJCOPY) -O binary --remove-section .bootloader $< $(call fixpath,${@})
```

File: Examples/MAX32655/BLE_otas/project.mk [hunk 1]
```diff
@@ -31,6 +31,8 @@ SRCS += stack_dats.c
 SRCS += dats_main.c
 SRCS += main.c
 
+PROJ_CFLAGS += -Wl,--section-start=.text=0x10004000
```
[the remark this answers is not in the document]
Comment (AbbyWolf-ADI) [reply Examples/MAX32655/BLE_otas/project.mk:34]: doesn't look like we do, I've removed it

File: Examples/MAX32655/BLE_otas/ota_internal_mem.ld [hunk 2]
```diff
@@ -46,12 +46,17 @@ MEMORY {
 }
 
 SECTIONS {
+    .bootloader :
+    {
+	KEEP(*bootloader.o)
+    }
```
Comment (kevin-gillespie) [inline Examples/MAX32655/BLE_otas/ota_internal_mem.ld:52]: We need the > BOOT here.

**Review by kevin-gillespie — CHANGES_REQUESTED** [review]
I think we accidentally added some documentation pictures and deleted one of the peripheral drivers.

**Review by EdwinFairchild — APPROVED** [review]

**Comment by EdwinFairchild** [comment]
Added a bug fix that this helped discover.

**Review by kevin-gillespie — APPROVED** [review]

**Pull request merged**